cmd/firesol/cli/node.go

```diff
@@ -198,29 +198,29 @@ func nodeFactoryFunc(app, kind string, appLogger *zap.Logger, appTracer logging.
 		if err != nil {
 			return nil, fmt.Errorf("unable to create chain operator: %w", err)
 		}
-		mergedBlocksStoreURL := MustReplaceDataDir(sfDataDir, viper.GetString("common-blocks-store-url"))
+		mergedBlocksStoreURL := MustReplaceDataDir(sfDataDir, viper.GetString("common-merged-blocks-store-url"))
 
-		var mindreaderPlugin *mindreader.MindReaderPlugin
+		var readerPlugin *mindreader.MindReaderPlugin
 		var registerServices func(server *grpc.Server) error
 
-		if kind == "mindreader" {
-			zlog.Info("preparing mindreader plugin")
+		if kind == "reader" {
+			zlog.Info("preparing reader plugin")
 			blockStreamServer := blockstream.NewUnmanagedServer(blockstream.ServerOptionWithLogger(appLogger))
-			oneBlockStoreURL := MustReplaceDataDir(sfDataDir, viper.GetString("common-oneblock-store-url"))
+			oneBlockStoreURL := MustReplaceDataDir(sfDataDir, viper.GetString("common-one-block-store-url"))
 
 			mergeThresholdBlockAge := viper.GetString(app + "-merge-threshold-block-age")
 			workingDir := MustReplaceDataDir(sfDataDir, viper.GetString(app+"-working-dir"))
 			blockDataWorkingDir := MustReplaceDataDir(sfDataDir, viper.GetString(app+"-block-data-working-dir"))
-			batchStartBlockNum := viper.GetUint64("mindreader-node-start-block-num")
-			batchStopBlockNum := viper.GetUint64("mindreader-node-stop-block-num")
-			blocksChanCapacity := viper.GetInt("mindreader-node-blocks-chan-capacity")
-			waitTimeForUploadOnShutdown := viper.GetDuration("mindreader-node-wait-upload-complete-on-shutdown")
-			oneBlockFileSuffix := viper.GetString("mindreader-node-oneblock-suffix")
-			batchFilePath := viper.GetString("mindreader-node-deepmind-batch-files-path")
-			purgeAccountChanges := viper.GetBool("mindreader-node-purge-account-data")
+			batchStartBlockNum := viper.GetUint64("reader-node-start-block-num")
+			batchStopBlockNum := viper.GetUint64("reader-node-stop-block-num")
+			blocksChanCapacity := viper.GetInt("reader-node-blocks-chan-capacity")
+			waitTimeForUploadOnShutdown := viper.GetDuration("reader-node-wait-upload-complete-on-shutdown")
+			oneBlockFileSuffix := viper.GetString("reader-node-oneblock-suffix")
```

Comment:
We miss this one to be standard on `firehose-ethereum`. Let's make sure all others use the name:
```suggestion
			oneBlockFileSuffix := viper.GetString("reader-node-one-block-suffix")
```